layers/+email/mu4e/packages.el

```diff
@@ -53,6 +70,5 @@
 
 (defun mu4e/post-init-org ()
   ;; load org-mu4e when org is actually loaded
-  (with-eval-after-load 'org (require 'org-mu4e nil 'noerror)))
-
-
+  (when (configuration-layer/layer-usedp 'org)
```

Comment:
Not needed as `post-init-org` will only be executed if `org` is loaded. So the previous code was fine.